packages/twenty-front/src/utils/isDefined.ts

```diff
@@ -1,3 +1,2 @@
-export const isDefined = <T>(
-  value: T | undefined | null,
-): value is NonNullable<T> => value !== undefined && value !== null;
+export const isDefined = <T>(value: T): value is NonNullable<T> =>
+  typeof value !== 'undefined' && value !== null;
```

Comment:
value !== undefined ?
Also, I think we should make the name more explicit: isDefinedAndNotNull

Reply:
+ add test to catch the 'undefined' mistake

Reply:
I changed it to use `isUndefined` and `isNull` from `@sniptt/guards` instead!
Also, I renamed `isDefined` to `isNonNullable`, as the return value is `value is NonNullable<T>`, `NonNullable` being defined as follows:
```
/**
 * Exclude null and undefined from T
 */
type NonNullable<T> = T & {};
```
And I changed the tests a little, though the `undefined` case was already included.